lib/ulimit/ulimit.go

```diff
@@ -17,7 +17,7 @@ var (
 	supportsFDManagement = false
 
 	// getlimit returns the soft and hard limits of file descriptors counts
-	getLimit func() (uint64, uint64, error)
+	GetLimit func() (uint64, uint64, error)
```

Comment:
I'd consider exposing a _separate_ `GetLimit` function instead of making this variable public.